orchestrator-salt/src/main/resources/salt-common/salt/monitoring/cron/monitoring_cert_check

```diff
@@ -1,2 +1,2 @@
 # generated by salt, do not edit - call script day at 01:00:00
-0 1 * * * root /opt/salt/scripts/cert-helper.sh -b /opt/cdp-request-signer/conf/cdp-request-signer,/opt/cdp-vmagent/conf/vmagent,/opt/blackbox_exporter/node_exporter,/opt/blackbox_exporter/blackbox_exporter -s cdp-node-exporter,cdp-blackbox-exporter,cdp-request-signer,cdp-vmagent
+0 1 * * * root /opt/salt/scripts/cert-helper.sh -b /opt/cdp-request-signer/conf/cdp-request-signer,/opt/cdp-vmagent/conf/vmagent,/opt/cdp-prometheus/cdp-prometheus,/opt/blackbox_exporter/node_exporter,/opt/blackbox_exporter/blackbox_exporter -s cdp-node-exporter,cdp-blackbox-exporter,cdp-request-signer,cdp-vmagent,cdp-prometheus
```

Comment:
wondering if could be a check cdp-vmagent or prometheus is used, or just remove cdp-vmagent, as that will be cleared anyway (so guessing that in 1 year cdp-vmagent won't be in the code, and credentials should be re-generated in 1 year earliest)

Reply:
Yes you are right. I will just remove vmagent from this file. Later it will be removed anyway.